atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -571,6 +575,25 @@ private Iterator<Map.Entry<Cell, byte[]>> getSortedColumnsLocalWrites(
                 cellComparator);
     }
 
+    @VisibleForTesting
+    static void reportExpectationsCollectedData(
+            ExpectationsAwareTransaction transaction, ExpectationsMetrics metrics, boolean isStillRunning) {
```

Comment:
Some remarks,

- This does not have to live here. We could put it elsewhere but I haven't thought of a good place
- This can't be in `ExpectationsAwareTransaction` (as a default method or static) because `ExpectationsMetrics` is not accessible from there (otherwise, circular dep & moving metrics definition to the api package is kind of yuck)
- This is a static because I wanted an instance of `ExpectationsAwareTransaction` because that's all what I need. It's also much easier to test since I don't have to do the dance of constructing `SnapshotTransaction`.